crates/teloxide-macros/src/command_attr.rs

```diff
@@ -119,3 +142,54 @@ fn is_command_attribute(a: &Attribute) -> bool {
         _ => false,
     }
 }
+
+fn is_command_description(attr: &Attribute) -> bool {
+    for token in attr.tokens.clone() {
+        if let TokenTree::Group(group) = token {
+            for token in group.stream() {
+                if let TokenTree::Ident(ident) = token {
+                    if ident == "description" {
+                        return true;
+                    }
+                }
+            }
+        }
+    }
+    false
+}
+
+fn parse_command_description(attr: &Attribute) -> Option<String> {
+    if is_command_attribute(attr) {
+        for token in attr.tokens.clone() {
+            if let TokenTree::Group(group) = token {
+                for token in group.stream() {
+                    if let TokenTree::Ident(ident) = token {
+                        if ident == "description" {
+                            for token in group.stream() {
+                                if let TokenTree::Literal(lit) = token {
+                                    let description = lit.to_string();
+                                    return Some(
+                                        lit.to_string().trim()[1..description.len() - 1]
+                                            .replace(r"\n", "\n"),
+                                    );
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    None
+}
+
+fn parse_doc_comment(attr: &Attribute) -> Option<String> {
+    #[allow(clippy::collapsible_match)]
+    if let syn::Meta::NameValue(syn::MetaNameValue { lit, .. }) = attr.parse_meta().ok()? {
+        if let syn::Lit::Str(s) = lit {
+            return Some(s.value().trim().replace(r"\n", "\n"));
```

Comment:
I'm confused, why do you need to do that replace?

Reply:
The output of `/// Hello\nNew Line` is "Hello\\nNew line", so I replaced `\\n` with a new line `\n`

Reply:
I don't agree with this interpretation. `\n` in the doc comment should be interpreted literally, as text. To force newline in description a

```rust
/// Hello
/// New Line
```

should be used instead (ideally we could maybe parse markdown, but that seems like too much...).